### ISSUE TITLE
resp['user_data'] does not seem to exist anymore

### ISSUE THREAD

**Issue by rh8056** [justmedude/pylotoncycle#11]
When running a short debug script:
```
import pylotoncycle

username = 'x'
password = 'x'

conn = pylotoncycle.PylotonCycle(username, password)
workouts = conn.GetRecentWorkouts(5)
workout = workouts[0]
```
I get the following:

```
 % python3 ./pelo_debug.py 
Traceback (most recent call last):
  File "./pelo_debug.py", line 12, in <module>
    conn = pylotoncycle.PylotonCycle(username, password)
  File "/usr/local/lib/python3.9/site-packages/pylotoncycle/pylotoncycle.py", line 31, in __init__
    self.login(username, password)
  File "/usr/local/lib/python3.9/site-packages/pylotoncycle/pylotoncycle.py", line 52, in login
    self.total_workouts = resp['user_data']['total_workouts']
KeyError: 'user_data'
```

Commenting out line 52 in pylotoncycle.py resolves the issue.  I'm not sure what else that impacts though.  It appears that the [user_data] branch is no longer in the /auth/login/ response.   It do see that the total_workouts still shows when using GetMe:

```
me = conn.GetMe()
print(me)
```

I've never submitted a github issue before, and my python is incredibly basic, so I'll leave it to you to resolve correctly!

**Comment by superm1** [comment]
I just submitted a PR with a similar change yesterday myself.
https://github.com/justmedude/pylotoncycle/pull/10

**Comment by justmedude** [comment]
fixed, and 0.5.1 has been released. Please report back and issues and thank you for your feedback
